english/eating-walls/find-a-toilet/door/pushed/pushed.md

```diff
@@ -1,14 +1,20 @@
 A tall thin man in jester's motley stands before you.
 He sighs and lights a cigarette.
 
-You look down and realize you have just wet your britches.  

```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.